shuup_tests/front/test_product_detail_context.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2019, Shoop Commerce Ltd. All rights reserved.
+#
+# This source code is licensed under the OSL-3.0 license found in the
+# LICENSE file in the root directory of this source tree.
+from __future__ import unicode_literals
+
+from decimal import Decimal
+
+import pytest
+import six
+import test_app
```

Comment:
what is this `test_app`?

Reply:
No idea. I will remove those and clean the imports.